pinot-controller/src/test/java/org/apache/pinot/controller/api/PinotSegmentRestletResourceTest.java

```diff
@@ -165,6 +165,30 @@ public void testSegmentCrcApi()
     checkCrcRequest(rawTableName, segmentMetadataTable, 9);
   }
 
+  @Test
+  public void testDeleteSegmentsWithTimeWindow()
```

Comment:
This test does not cover for a particular window but dropping all segments right? Should we rename it accordingly or enhance the test?

Reply:
The endpoint is different with the one that drop all segments. ControllerTest.sendDeleteRequest(urlBuilder.forSegmentDeleteAPI(rawTableName)) will return /segments/<table>/select. The normal endpoint to drop all segments is /segments/<table>

The reason why I don't specify the time range is because mockSegmentMetadata hard code the time range to 1L-10L.  If the time window option is not claimed, the start and end will be filled with default value [0, +inf]. 

I can specify time range [0, Long.MAX_VALUE].

Reply:
I added more test cases.